docs/day-2-operations/Backups.adoc

```diff
@@ -144,8 +144,99 @@ Backup the dump.rb file from redis
 oc cp $(oc get pods -l 'deploymentConfig=system-redis' -o json | jq '.items[0].metadata.name' -r):/var/lib/redis/data/dump.rdb ./system-redis-dump.rdb
 ----
 
+=== `Secrets`
+[source,bash]
+----
+oc get secrets system-smtp -o json --export > system-smtp.json
+oc get secrets system-seed -o json --export > system-seed.json
+oc get secrets system-database -o json --export > system-database.json
+oc get secrets backend-internal-api -o json --export > backend-internal-api.json
+oc get secrets system-events-hook -o json --export > system-events-hook.json
+oc get secrets system-app -o json --export > system-app.json
+oc get secrets system-recaptcha -o json --export > system-recaptcha.json
+oc get secrets system-redis -o json --export > system-redis.json
+oc get secrets zync -o json --export > zync.json
+oc get secrets system-master-apicast -o json --export > system-master-apicast.json
+----
+
+=== `ConfigMaps`
+[source,bash]
+----
+oc get configmaps system-environment -o json --export > system-environment.json
+oc get configmaps apicast-environment -o json --export > apicast-environment.json
+----
+
 == Restore Procedures
 
+=== `Template-based deployment`
+
+[source,bash]
+
+Template parameters will be read from copied secrets and configmaps.
+
+----
+oc new-app --file /opt/amp/templates/amp.yml \
```

Comment:
is this sufficient? for example `system-smtp.json` isn't used here, `oc apply` might be relevant also for template-based restore

Reply:
Good catch! I think this is the only one not used. I will fix it